core/src/main/scala/org/bitcoins/core/protocol/dlc/CETCalculator.scala

```diff
@@ -369,23 +392,34 @@ object CETCalculator {
       totalCollateral: Satoshis,
       rounding: RoundingIntervals,
       min: Long,
-      max: Long): Vector[(Vector[Int], Satoshis)] = {
+      max: Long): Vector[CETOutcome] = {
     val ranges = splitIntoRanges(min, max, totalCollateral, function, rounding)
 
     ranges.flatMap { range =>
       range match {
         case StartZero(indexFrom, indexTo) =>
-          groupByIgnoringDigits(indexFrom, indexTo, base, numDigits).map(
-            _ -> Satoshis.zero)
+          groupByIgnoringDigits(indexFrom, indexTo, base, numDigits).map {
+            decomp =>
+              val payout = Satoshis.zero
+              CETOutcome(decomp, payout)
```

Comment:
nit: On all of these I prefer the one liner `CETOutcome(decomp, payout = Satoshis.zero)`